general/administration/hardware-acceleration.md

```diff
@@ -220,10 +220,12 @@ Prerequisites:
 
 Follow the instructions in the link above to install the NVIDIA Container Toolkit for your Linux distribution.
 
-Start your container by adding this parameter:
+Start your container by adding these parameters:
 
 ```sh
 --gpus all \
+-e NVIDIA_DRIVER_CAPABILITIES=all \
```

Comment:
This is already included
https://github.com/jellyfin/jellyfin-metapackages/blob/e78e91625c1292c86d8a604827f6acccb5b77f61/Dockerfile.amd64#L23

Reply:
Dropped in favour of <https://github.com/jellyfin/jellyfin-metapackages/pull/22>